test/numa/run_tests.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+TESTS_DIR="$1"
+
+usage() {
+    echo "Usage: run_tests.sh TESTS_DIR"
+    echo "TESTS_DIR is expected to be structured as POLICY/TOPOLOGY/TEST with files:"
+    echo "POLICY/cri-resmgr.cfg: configuration of cri-resmgr"
+    echo "POLICY/TOPOLOGY/topology.var.json: contents of the topology variable for run.sh"
+    echo "POLICY/TOPOLOGY/code.var.sh: contents of the code var (that is, test script)"
+}
+
+error() {
+    (echo ""; echo "error: $1" ) >&2
+    exit 1
+}
+
+export_var_files() {
+    local var_file_dir="$1"
+    local var_filepath
+    local var_file_name
+    local var_name
+    for var_filepath in "$var_file_dir"/*.var "$var_file_dir"/*.var.*; do
+        if ! [ -f "$var_filepath" ] || [[ "$var_filepath" == *"~" ]]; then
+            continue
+        fi
+        var_file_name=$(basename "$var_filepath")
+        var_name=${var_file_name%%.var*}
+        echo "exporting $var_name from $var_filepath"
+        export "$var_name"="$(< "$var_filepath")"
+    done
+}
+
+if [ -z "$TESTS_DIR" ] || [ "$TESTS_DIR" == "help" ] || [ "$TESTS_DIR" == "--help" ]; then
+    usage
+    error "missing TESTS_DIR"
+fi
+
+if ! [ -d "$TESTS_DIR" ]; then
+    error "bad TESTS_DIR: \"$TESTS_DIR\""
+fi
+
+for POLICY_DIR in "$TESTS_DIR"/*; do
+    if ! [ -d "$POLICY_DIR" ]; then
+        continue
+    fi
+    for CFG_FILE in "$POLICY_DIR"/*.cfg; do
+        if ! [ -f "$CFG_FILE" ]; then
+            error "cannot find cri-resmgr configuration $POLICY_DIR/*.cfg"
+        fi
+        export cri_resmgr_cfg=$CFG_FILE
+    done
+    export_var_files "$POLICY_DIR"
+    for TOPOLOGY_DIR in "$POLICY_DIR"/*; do
+        if ! [ -d "$TOPOLOGY_DIR" ]; then
+            continue
+        fi
+        export_var_files "$TOPOLOGY_DIR"
+        vm="$(basename "$TOPOLOGY_DIR")"
+        export vm
+        for TEST_DIR in "$TOPOLOGY_DIR"/*; do
+            if ! [ -d "$TEST_DIR" ]; then
+                continue
+            fi
+            export_var_files "$TEST_DIR"
+            export outdir="$TEST_DIR/output"
+            mkdir -p "$outdir"
+            ./run.sh test 2>&1 | tee "$outdir/run.sh.output"
```

Comment:
This is the one to get replaced with `$RUN_SH test ...` mentioned in the previous comment.